app/api/event_invoices.py

```diff
@@ -136,7 +139,18 @@ class EventInvoiceDetail(ResourceDetail):
                   'model': EventInvoice}
 
 
-class EventInvoiceRelationship(ResourceRelationship):
+class EventInvoiceRelationshipRequired(ResourceRelationship):
+    """
+    Event Invoice Relationship
+    """
+    decorators = (is_admin,)
```

Comment:
I think it is good to use permission manager to maintain uniformity and we can remove those duplicate decorators in next PRs.